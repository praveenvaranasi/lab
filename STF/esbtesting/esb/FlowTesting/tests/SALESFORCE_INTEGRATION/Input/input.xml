<ns1:SQL_CFG_1 xmlns:ns1="http://www.fiorano.com/fesb/activity/DB2/Request" id="-9223372036854775808">
   <ns1:EnterProductDetails>
      <ns1:TIF_RECORDID>1000</ns1:TIF_RECORDID>
      <ns1:FIRSTNAME>FN1</ns1:FIRSTNAME>
      <ns1:LASTNAME>LN1</ns1:LASTNAME>
      <ns1:EMAILID>praveenk434@gmail.com</ns1:EMAILID>
      <ns1:PRODUCTPURCHASED>fsoa</ns1:PRODUCTPURCHASED>
      <ns1:AMOUNTPAID>50000</ns1:AMOUNTPAID>
      <ns1:TIF_STATUS>N</ns1:TIF_STATUS>
   </ns1:EnterProductDetails>
   <ns1:EnterProductDetails>
      <ns1:TIF_RECORDID>1001</ns1:TIF_RECORDID>
      <ns1:FIRSTNAME>FN2</ns1:FIRSTNAME>
      <ns1:LASTNAME>LN2</ns1:LASTNAME>
      <ns1:EMAILID>krishna.aditya@fiorano.com</ns1:EMAILID>
      <ns1:PRODUCTPURCHASED>fmq</ns1:PRODUCTPURCHASED>
      <ns1:AMOUNTPAID>40000</ns1:AMOUNTPAID>
      <ns1:TIF_STATUS>N</ns1:TIF_STATUS>
   </ns1:EnterProductDetails>
</ns1:SQL_CFG_1>
