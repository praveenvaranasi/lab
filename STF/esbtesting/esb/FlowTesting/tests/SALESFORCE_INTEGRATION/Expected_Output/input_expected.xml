<?xml version="1.0" encoding="UTF-8"?><tns:Envelope xmlns:tns="urn:enterprise.soap.sforce.com/convertLead/Output" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns:soapenc="http://schemas.xmlsoap.org/soap/encoding/" xmlns:xsd="http://www.w3.org/2001/XMLSchema" xmlns:soapenv="http://schemas.xmlsoap.org/soap/envelope/">
   <tns:Header/><tns:Body>
   <convertLeadResponse xmlns="urn:enterprise.soap.sforce.com"><result><accountId>0017000000MIBXBAA5</accountId><contactId>0037000000TkajsAAB</contactId><errors xsi:nil="true"/><leadId>00Q7000000IYJm1EAH</leadId><opportunityId>0067000000AiyaXAAR</opportunityId><success>true</success></result></convertLeadResponse></tns:Body>
</tns:Envelope>
